deploy/twingate-operator/values.yaml

```diff
@@ -22,6 +22,22 @@ twingateOperator: {}
 #    tag1: value_for_tag1
 #    tag2: value_for_tag2
 
+## Restrict operator to monitor resources in specific namespaces. You should either:
+## 1. Specify a list of namespaces:
+## namespaces: ["foo-dev", "foo-stg"]
+## 2. Use globs to match multiple namespaces:
+## namespaces: ["*-dev", "*-stg"]
+## 3. Use negation to include all namespaces except those excluded:
+## namespaces: ["!*-test"]
+## 4. Use multiple globs in one pattern:
+## namespaces: ["foo-*,!*-test"]
+## For more information on using multiple globs, see: https://kopf.readthedocs.io/en/stable/scopes
```

Comment:
> Multiple globs can be used in one pattern. The rightmost matching one wins. The first glob is decisive: if a namespace does not match it, it does not match the whole pattern regardless of what is there (other globs are not checked). If the first glob is a negation, it is implied that initially, all namespaces do match (as if preceded by *), and then the negated ones are excluded. 

This is the [full explanation](https://kopf.readthedocs.io/en/stable/scopes/#namespaces) of how the comma-separated globs pattern works. I didn't include it here because it's very long and overwhelming. I think in most cases, methods 1-3 should be sufficient.